tsconfig.json

```
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "checkJs": true,
+    "noEmit": true,
+    "target": "esnext",
```

Thanks for the super-fast response, no rush on my part. Here is the suggested change in case you want to use it:

```suggestion
    "target": "es5",
```